provider/sgx.py

```diff
@@ -148,8 +148,11 @@ def verify_qmp_host_sgx_cap(self, host_epc_size):
         host_sgx_info = self._monitor.query_sgx_capabilities()
         self.verify_sgx_flags(host_sgx_info, sgx_flags)
 
-        host_qmp_section_size = host_sgx_info['section-size']
-        if host_epc_size != int(host_qmp_section_size):
+        host_qmp_sections = host_sgx_info['sections']
```

Comment:
hi @zixi-chen , one concern from my side, if we run this case against a qemu build whose version is less than 8.0, we could get a failure because the old qemu still uses section-size instead of sections in qemu8, so do we need to consider this?

If so, we may introduce a _get_section_size(sgx_info) function to get the section size, if section-size in sgx_info, that's an old qemu, we just return int(sgx_info['section-size'], or we use sections to make a sum and return it.

What do you think?

Reply:
qmp sections info with size on each node is not new in qemu 8, so we don't need to worry about test cases failing before qemu8.0.

Reply:
So before 8.0, we can also get the section size with 'sections', right?

Reply:
Yes.

Reply:
Thanks Zixi for the explanation, it's clear.